src/domain/operation_mode.py

```diff
@@ -119,6 +120,12 @@ def send_email(self, body, img_path):
             smtp_server.quit()
         logger.info("Email notification for %s sent!", img_path)
 
+    def actuate(self):
+        """Method to trigger actuators when enabled"""
+        if actuators:
+            # TODO: implement actuator logic
+            pass
```

Comment:
Isn't it better to generate an exception to intercept those case?
If the code needs to work now even without actuators, you can leave empty the method and implement the code later.

Reply:
We're gonna integrate Antonio's logic today.. this is just a skeleton to drive him